Automattic/jetpack
# VideoPress: tweak Remove poster button

~Follow-up and branched off from of VideoPress: add Poster panel to video block sidebar #29150~

Related to https://github.com/Automattic/jetpack/issues/28781

Fixes #

## Proposed changes:

* VideoPress: Tweak the Remove poster button styles

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add/edit a VideoPress video block
* Confirm the poster feature keeps working as expected
* Confirm the visual changes

before | after
------|-----
<img width="265" alt="image" src="https://user-images.githubusercontent.com/77539/221250401-4176ba87-941b-43b7-b04e-bdbca14eb4c7.png"> | <img width="294" alt="image" src="https://user-images.githubusercontent.com/77539/221249403-78dfe6d7-54b5-4330-8839-783a03d0f6ac.png">

before | after
------|-----
<img width="286" alt="image" src="https://user-images.githubusercontent.com/77539/221250352-5f33d021-5161-4461-b7b4-0e6a71f9b0b5.png"> | <img width="298" alt="image" src="https://user-images.githubusercontent.com/77539/221249457-bbdc4092-c11c-45e8-87d5-a68243bb4cf6.png">
